### PR TITLE
Follow up on CF Bump to 3.6.0. Addressing remaining comments.

### PR DESCRIPTION
**Pull request by andreas-kupries**
- Remove various superfluous quotes in yaml values.
- Bump the CF_VERSION reported by scf.
- Added comments to the bpm patch scripts explaining our need for 'unsafe.unrestricted_volumes'.

See https://github.com/SUSE/scf/pull/1921 for the origin of the comments fixed here.
Ref https://trello.com/c/wqcKOPJC/895-cf-bump-part-1-cf-v36

### CODE REVIEW

File: container-host-files/etc/scf/config/scripts/patches/fix_route_registrar_bpm_dns.sh [hunk 1]
```diff
@@ -17,6 +17,20 @@ if [ -f "${SENTINEL}" ]; then
...
 fi
 
+# Notes on "unsafe.unrestricted_volumes":
+#
+# - The mounts 1, 2, and 4 are required to make DNS work in the nested
+#   container created by BPM for the job to run in.
+#
+# - The remainer are required to give the job access to the system
+#   root certificates so that it actually can verify the certs given
+#   to it by its partners.
```
Comment (mook-as) [inline container-host-files/etc/scf/config/scripts/patches/fix_route_registrar_bpm_dns.sh]: This one has all of `/var/lib`, not just `/var/lib/ca-certificates`; is that intentional?

File: container-host-files/etc/scf/config/scripts/patches/fix_routingapi_bpm_dns.sh [hunk 2]
```diff
@@ -29,6 +29,15 @@ patch -d "$PATCH_DIR" --force -p0 <<'PATCH'
...
 
+# Notes on "unsafe.unrestricted_volumes":
+#
+# - The mounts 1, 2, and 4 are required to make DNS work in the nested
+#   container created by BPM for the job to run in.
+#
+# - The remainer are required to give the job access to the system
+#   root certificates so that it actually can verify the certs given
+#   to it by its partners.
+
```
Comment (mook-as) [inline container-host-files/etc/scf/config/scripts/patches/fix_routingapi_bpm_dns.sh]: This one also has all of `/var/lib`; please confirm that is desired.

File: container-host-files/etc/scf/config/scripts/patches/fix_route_registrar_bpm_dns.sh (hunk 1 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/scripts/patches/fix_route_registrar_bpm_dns.sh]: Likely Jan simply missed it when he did the work ... I will make the fix now. And fix the typo I just saw.

File: container-host-files/etc/scf/config/scripts/patches/fix_routingapi_bpm_dns.sh (hunk 2 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/scripts/patches/fix_routingapi_bpm_dns.sh]: S.a.

File: container-host-files/etc/scf/config/scripts/patches/fix_route_registrar_bpm_dns.sh (hunk 1 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/scripts/patches/fix_route_registrar_bpm_dns.sh]: I also note leftover `/etc/pki` now. Will be removed.

Comment (andreas-kupries) [reply container-host-files/etc/scf/config/scripts/patches/fix_route_registrar_bpm_dns.sh]: Done with commit 545bd60f8d80c04ed088d12aed2717db83377302

File: container-host-files/etc/scf/config/scripts/patches/fix_routingapi_bpm_dns.sh (hunk 2 above)
Comment (andreas-kupries) [reply container-host-files/etc/scf/config/scripts/patches/fix_routingapi_bpm_dns.sh]: Done with commit 545bd60f8d80c04ed088d12aed2717db83377302

**Pull request merged**